src/containers/Profile/ProfileInfo/index.tsx

```diff
@@ -49,11 +49,11 @@ const ProfileInfo: FC<ComponentProps> = ({github_username}) => {
             </div>
           ))}
         <div className="ProfileInfo__member-slack">
-          <Icon icon={IconType.slack} onClick={() => null} size={18} />
+          <Icon className="ProfileInfo__member-slack-icon" icon={IconType.slack} onClick={() => null} size={18} />
```

Comment:
Hey I wanted to know the reason to pass `()=>null` to `onClick` here ?

Reply:
Opps my bad forgot to remove this